components/kyma-environment-broker/internal/avs/model.go

```diff
@@ -130,7 +130,7 @@ func generateNameAndDescription(operation internal.ProvisioningOperation, beType
 
 func providerCodeByPlan(planID string) string {
 	switch planID {
-	case broker.AWSPlanID:
+	case broker.AWSPlanID, broker.AWSHAPlanID:
```

Comment:
please, create a ticket to use CloudProvider field instead of duplicating a logic